portal/templates/contract/components/clone-deal-link.njk

```diff
@@ -8,8 +8,9 @@
         href="/contract/{{ params.deal._id }}/clone/before-you-start"
         class="govuk-link govuk-!-font-size-14"
         data-cy="clone-deal-link"
-        aria-label="Clone Deal"
-      >Clone</a>
+        title="Clone Deal {{ params.deal.bankInternalRefName }}"
+        aria-label="Clone Deal {{ params.deal.bankInternalRefName }}"
+      >Clone {{ params.deal.bankInternalRefName }}</a>
```

Comment:
same as above

Reply:
also, is this applicable to both BSS and GEF deals?

Reply:
sent what it looks like on slack.  And currently only on BSS.  Not got the tickets for GEF analysis yet